x/epoching/spec/README.md

```diff
@@ -0,0 +1,37 @@
+<!--
+order: 20
+title: Epoching Overview
+parent:
+  title: "epoching"
+-->
+
+# `x/epoching`
```

Comment:
We'll need to expand upon this a bit. Maybe include a few very relevant APIs/methods.

Reply:
ill add examples as the other prs land.